helpers/mathjax.js

```diff
@@ -182,12 +182,15 @@ export function loadMathJax(mathJaxConfig) {
 				//  renders the document.  The MathDocument is returned in case
 				//  you need to rerender the shadowRoot later.
 				//
-				window.MathJax.typesetShadow = function(root) {
-					const InputJax = startup.getInputJax();
-					const OutputJax = startup.getOutputJax();
-					const html = mathjax.document(root, { InputJax, OutputJax });
-					html.render().typeset();
-					return html;
+				window.MathJax.typesetShadow = async function(root) {
+					return await mathjax.handleRetriesFor(() => {
```

Comment:
This is basically how MathJax handles missing extensions. It tries to render and if it hits a missing extension, it loads it and triggers a retry. Because we're not using MathJax's built-in adaptor, we have to do this ourselves.